Sources/Workspace/InitPackage.swift

```diff
@@ -498,3 +499,13 @@ extension SupportedPlatform {
         }
     }
 }
+
+private func getDefaultPlatforms() -> [SupportedPlatform] {
+    #if os(macOS)
```

Comment:
I am not sure about this. most swift developers develop on a fairly recent Mac regardless on what the target deployment platforms are

Reply:
are you commenting on the functionality in general or the `#if` here? if you mean the functionality in general should that be discussed on the pitch? https://forums.swift.org/t/pitch-generate-package-swift-files-with-newer-macos-versions/61925

I just don't want to dup the justifications here if that answers some of these